lib/display/display.ex

```diff
@@ -83,4 +96,32 @@ defmodule Inky.Display do
     # Little endian, unsigned short
     <<rows::unsigned-little-integer-16>>
   end
+
+  # colorsets from pimoroni library
+  # https://github.com/pimoroni/inky/blob/54684464b2f35bfd52208cdfb922c09685644181/library/inky/inky_uc8159.py#L26-L46
+  defp get_colorset(:desaturated) do
+    [
+      [0, 0, 0],
+      [255, 255, 255],
+      [0, 255, 0],
+      [0, 0, 255],
+      [255, 0, 0],
+      [255, 255, 0],
+      [255, 140, 0],
+      [255, 255, 255]
+    ]
+  end
+
+  defp get_colorset(:saturated) do
+    [
+      [57, 48, 57],
+      [255, 255, 255],
+      [58, 91, 70],
+      [61, 59, 94],
+      [156, 72, 75],
+      [208, 190, 71],
+      [177, 106, 73],
+      [255, 255, 255]
+    ]
+  end
```

Comment:
Feels like things are getting  muddled in display.ex... we might want to consider how things could be split up. Not sure if it should be by amount of colours or "base type" of display. The what/phat share things, but I'm seeing quite a few things that aren't really compatible with the impression.